storage/src/checker/mod.rs

```diff
@@ -256,41 +259,44 @@ mod test {
 
     // set up a basic trie:
     // -------------------------
+    // | partial path: []      |
     // |     |  X  |  X  | ... |    Root node
     // -------------------------
     //    |
     //    V
     // -------------------------
-    // |  X  |     |  X  | ... |    Branch node
+    // | partial path: [9]     |
+    // |  X  |     |  X  | ... |    Branch node (path: 0x09)
```

Comment:
Why are we changing this in this PR?

This feels really random. Is that the intention?

Seems like having something more orthogonal would make understanding this structure a bit easier going forward.